docs/user/exercises/programming.rst

```diff
@@ -101,6 +101,62 @@ The online editor displays only visible files, to avoid showing for example ``.g
 This also means that other config files, like SwiftLint's ``.swiftlint.yml`` file, are not shown. Those files currently
 can only be accessed via your own IDE.
 
+Manual Assessment
+-----------------
+
+There are two ways to manually assess programming exercises: with the Orion plugin, you can assess the student's code directly in your JetBrains IDE. Refer to the :ref:`Orion plugin documentation <orion>` for more information.
+Alternatively, you can use the online editor to assess the students' submissions.
+
+.. figure:: programming/online-editor-assessment.png
+   :alt: Manual assessment in the online editor
+   :align: center
+
+   Manual assessment in the online editor
+
+For manual assessment, the online editor provides the following features, tailored to the assessment process:
+
+1. **File browser**: The file browser shows the files in the student's submission and allows you to navigate between them. Changed files are highlighted.
+2. **Build output**: The build output shows the output of the build process, if any. This is useful in case of build errors.
```

Comment:
```suggestion
2. **Build output**: The build output shows the output of the build process, if any exists. This is useful in case of build errors.
```